src/shared/views/Project.tsx

```diff
@@ -168,26 +168,39 @@ const ProjectView: React.FunctionComponent<ProjectViewProps> = ({
   );
 };
 
-const mapStateToProps = (state: RootState) => ({
-  authenticated: !!state.oidc.user,
-  project:
+const mapStateToProps = (state: RootState) => {
+  const orgData =
+    state.nexus && state.nexus.activeOrg && state.nexus.activeOrg.data;
+  const projectData =
     (state.nexus &&
       state.nexus.activeProject &&
-      state.nexus.activeProject.data &&
       state.nexus.activeProject.data) ||
-    null,
-  isFetching:
-    (state.nexus &&
-      state.nexus.activeProject &&
-      state.nexus.activeProject.isFetching) ||
-    true,
-  error:
-    (state.nexus &&
-      state.nexus.activeProject &&
-      state.nexus.activeProject.error) ||
-    null,
-  lists: state.lists || {},
-});
+    null;
+
+  const orgLabel = (orgData && orgData.org.label) || '';
+  const projectLabel = (projectData && projectData.label) || '';
+  const activeListId =
+    (state.lists &&
+      Object.keys(state.lists).find(key => key === orgLabel + projectLabel)) ||
```

Comment:
They should be joined with a kind of separator so that there cannot be collisions if an orgLabel is the same than another orgLabel+projectLabel

Reply:
I don't really want to change too much logic, it's a little brittle plus a list only belongs to a project, that belongs to an org. And you can't have 2 orgs with the same name, so `org`+`project` will always be unique to the specific project